### ISSUE TITLE
BindableLayout width

### ISSUE THREAD

**Issue by DJafari** [mrmans0n/smart-adapters#12]
on some device, such as huawei U20, layout width is wrap_content ( see below picture )

![device-2015-09-27-141353](https://cloud.githubusercontent.com/assets/1963047/10122320/1711b158-6522-11e5-99c0-d7bc8a000418.png)

**Comment by DJafari** [comment]
after this update, for listview return an exception :

```
java.lang.ClassCastException: android.widget.FrameLayout$LayoutParams cannot be cast to android.widget.AbsListView$LayoutParams
```

**Comment by mrmans0n** [comment]
Ok.

**Comment by mrmans0n** [comment]
This should be fixed in version 1.3.0, but you will have to specify your layout_width on your own for this to work. As the LayoutParms depend always on the parent view, you can use that to your advantage in your views now and do it programmatically. I have added some examples in the code and in the README for this.
